src/slic3r/GUI/GUI_Init.cpp

```diff
@@ -36,6 +36,10 @@ int GUI_Run(GUI_InitParams &params)
     signal(SIGCHLD, SIG_DFL);
 #endif // __APPLE__
 
+#ifdef __FreeBSD__
+    wxDISABLE_DEBUG_SUPPORT();
```

Comment:
I'm not actually sure why others aren't seeing all the wxASSERT() dialogs, but this suppresses them for me.  No good reason to make this #ifdef __FreeBSD__, but I didn't want to change desired behaviour (but did want to stop the assert dialogs for Release builds).